python/pipeline/meso.py

```diff
@@ -519,55 +565,301 @@ def make(self, key):
             "fill_fraction": fill_fraction,
             "template": template,
         }
+
+        # Determine border cutoff to avoid edge artifacts
+        px_height, px_width = (ScanInfo.Field & key).fetch1("px_height", "px_width")
+        skip_rows = int(round(px_height * 0.10))
+        skip_cols = int(round(px_width * 0.10))
+
+        # Determine xy shifts via phase correlation
```

Comment:
could you add to this comment an explanation for why the field id and channel id variables have dummy entries?